VMBackup/main/hostsnapshotter.py

```diff
@@ -156,7 +149,20 @@ def pre_snapshot(self, paras, taskId, fetch_disk_details = False):
                 self.logger.log("start calling the presnapshot rest api")
                 # initiate http call for blob-snapshot and get http response
                 result, httpResp, errMsg,responseBody = http_util.HttpCallGetResponse('POST', presnapshoturi_obj, body_content, headers = headers, responseBodyRequired = True, isHostCall = True)
-                self.logger.log("presnapshot responseBody: " + responseBody)
+                if responseBody:
+                    try:
+                        response_json = json.loads(responseBody)
+                        if "bhsVersion" in response_json:
+                            self.logger.log("PreSnapshotResponse: bhsVersion: " + str(response_json["bhsVersion"]))
+                        if "nodeId" in response_json:
+                            self.logger.log("PreSnapshotResponse: nodeId: " + str(response_json["nodeId"]))
+                        if "responseTime" in response_json:
+                            self.logger.log("PreSnapshotResponse: responseTime: " + str(response_json["responseTime"]))
+                        if "result" in response_json:
+                            self.logger.log("PreSnapshotResponse: result: " + str(response_json["result"]))
```

Comment:
will result carry encryption details? If so, let us not log result, rather we can log if result was not null

Reply:
The result just would have success or failure, they would not be having the encryption details